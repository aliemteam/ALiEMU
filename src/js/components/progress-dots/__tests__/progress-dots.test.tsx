import React from 'react';
import renderer from 'react-test-renderer';

import ProgressDots from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<ProgressDots steps={5} currentStep={3} />);
    expect(tree).toMatchSnapshot();
});
