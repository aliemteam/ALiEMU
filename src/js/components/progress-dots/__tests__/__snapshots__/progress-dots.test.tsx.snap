// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  role="tablist"
>
  <button
    aria-selected={false}
    className=""
    data-step={0}
    onClick={[Function]}
    role="tab"
    tabIndex={-1}
    type="button"
  />
  <button
    aria-selected={false}
    className=""
    data-step={1}
    onClick={[Function]}
    role="tab"
    tabIndex={-1}
    type="button"
  />
  <button
    aria-selected={false}
    className=""
    data-step={2}
    onClick={[Function]}
    role="tab"
    tabIndex={-1}
    type="button"
  />
  <button
    aria-selected={true}
    className="undefined"
    data-step={3}
    onClick={[Function]}
    role="tab"
    tabIndex={-1}
    type="button"
  />
  <button
    aria-selected={false}
    className=""
    data-step={4}
    onClick={[Function]}
    role="tab"
    tabIndex={-1}
    type="button"
  />
</div>
`;
